package.json

```diff
@@ -22,8 +22,15 @@
     "dist/*.{js,css,map}"
   ],
   "dependencies": {
-    "@creativecommons/fonts": "^2020.11.3",
-    "@creativecommons/vocabulary": "^2020.11.3",
+    "@creativecommons/cc-assets": "^0.1.0",
+    "@creativecommons/vocabulary-fonts": "^0.1.1",
+    "@creativecommons/vocabulary-styles": "^0.2.1",
+    "@fortawesome/fontawesome-svg-core": "^1.3.0-beta3",
+    "@fortawesome/free-brands-svg-icons": "^5.15.4",
+    "@fortawesome/free-regular-svg-icons": "^6.0.0-beta3",
+    "@fortawesome/free-solid-svg-icons": "^6.0.0-beta3",
+    "@fortawesome/react-fontawesome": "^0.1.16",
```

Comment:
Do we need react-fontawesome?

Reply:
No I don't think so. 
Let me remove that.